errors.go

```diff
@@ -22,6 +22,9 @@ import (
 
 // common error objects used for argument checking
 var (
+	ErrNeedPod         = errors.New("Pod must be specified")
 	ErrNeedPodID       = errors.New("Pod ID cannot be empty")
 	ErrNeedContainerID = errors.New("Container ID cannot be empty")
+	ErrNeedFile        = errors.New("file cannot be empty")
```

Comment:
@dlespiau - interesting, but looking at the virtcontainers code:

```
$ git grep -p 'fmt.Errorf.*"[A-Z]' | wc -l
220
$ git grep -p 'fmt.Errorf.*"[a-z]' | wc -l
40
```

Maybe an opportunity here to write a new linter for `Error`'s to plug into https://github.com/alecthomas/gometalinter? :smile: